Wikipedia/Code/DiffListContextViewModel.swift

```diff
@@ -107,10 +107,11 @@ final class DiffListContextViewModel: DiffListGroupViewModel {
     private(set) var expandedHeight: CGFloat = 0
     private(set) var innerPadding: NSDirectionalEdgeInsets
     
-    static let contextItemTextPadding = NSDirectionalEdgeInsets(top: 3, leading: 8, bottom: 8, trailing: 8)
+    static let contextItemTextPadding = NSDirectionalEdgeInsets(top: 15, leading: 8, bottom: 0, trailing: 8)
     static let contextItemStackSpacing: CGFloat = 5
     static let containerStackSpacing: CGFloat = 15
-    
+    private(set) var headerHeight: CGFloat = 35
```

Comment:
I'm curious about the intention of this property – it may be contributing to clipping I'm occasionally noticing with the first compressed element.

![clip](https://github.com/wikimedia/wikipedia-ios/assets/5652327/381c5e6c-6ad9-4a54-b995-269c90e775ec)

Reply:
Setting a header height was needed so the div view had the correct spacing to the text, adding it through only a constraint and not adjusting this height caused the cell height to be calculated incorrectly and hide the disclosure button. That new cell height might have thrown things off on this view, I'll take a look at that as well

Reply:
Made an adjustment to that text clipping